### DISCUSSION TITLE
Unable to configure TIPTAP PRO TOKEN with Vercel

### DISCUSSION THREAD

**Discussion by ccreusat** [vercel/community#5117] [Help]
### Summary

Hello,

I'm trying to set up TIPTAP PRO TOKEN with Vercel to install the mathematics extension.

I followed the doc : https://vercel.com/guides/using-private-dependencies-with-vercel#other-package-registries
But it's not working because I have this error during install : 

```
WARN  GET https://registry.tiptap.dev/@tiptap-pro/extension-mathematics/-/extension-mathematics-2.2.1.tgz error (ERR_PNPM_FETCH_500). Will retry in 10 seconds. 2 retries left.
```

This is how my `.npmrc` file is and how I configure it with Vercel CLI : 
 
```
@tiptap-pro:registry=https://registry.tiptap.dev/
//registry.tiptap.dev/:_authToken=TIPTAP_PRO_TOKEN
```

Then

```
vercel env add NPM_RC preview < .npmrc
```

I also have the `TIPTAP_PRO_TOKEN` configured in environment-variables settings

What am I doing wrong, what am I missing ?

### Steps to Reproduce

- Have a `.npmrc` file gitignored with the following configuration 

```
@tiptap-pro:registry=https://registry.tiptap.dev/
//registry.tiptap.dev/:_authToken=TIPTAP_PRO_TOKEN
```

- Push the configuration with Vercel CLI : 
```
vercel env add NPM_RC preview < .npmrc
```

- Add TIPTAP PRO TOKEN in the environment variable settings.

`TIPTAP_PRO_TOKEN` is the key, then add the token as value

- Try to deploy to launch install & build

**Comment by jonahallibone** [comment]
Can we reopen this? Also facing this challenge

**Comment by mason-os** [comment]
`.npmrc` files require environment variables to be wrapped in curly braces (like in bash strings), e.g. `${TIPTAP_PRO_TOKEN}`. My `.npmrc` ends up being:
```
@tiptap-pro:registry=https://registry.tiptap.dev/
//registry.tiptap.dev/:_authToken=${TIPTAP_PRO_TOKEN}
```

And this works for me with the `TIPTAP_PRO_TOKEN` environment variable set in Vercel.

[a comment by an automated account is not shown]
